src/components/title/show/package-filter-modal/package-filter-modal.js

```diff
@@ -93,27 +129,50 @@ const PackageFilterModal = ({
         onClose={toggleModal}
         id="package-filter-modal"
       >
-        <Label id={labelId}>
-          <FormattedMessage id="ui-eholdings.label.packages" />
-        </Label>
-        <MultiSelection
-          autoFocus
-          dataOptions={allOptions}
-          onChange={handleFilterChange}
-          value={selectedOptions}
-          data-test-package-filter-select
-          id="packageFilterSelect"
-          ariaLabelledBy={labelId}
-          aria-label={<FormattedMessage id="ui-eholdings.label.packages" />}
-          tether={{
-            constraints: [
-              {
-                to: 'window',
-                attachment: 'together',
-              }
-            ]
-          }}
-        />
+        <div role="tablist">
+          <Accordion
+            id={labelId}
+            name={labelId}
+            label={<FormattedMessage id="ui-eholdings.label.packages" />}
+            separator={false}
+            closedByDefault={false}
+            header={FilterAccordionHeader}
+            displayClearButton={selectedOptions?.length}
+            onClearFilter={clearSelectedOptions}
+            headerProps={{
+              headingLevel: 2,
+              role: 'tab',
+            }}
+          >
+            <MultiSelection
+              autoFocus
+              dataOptions={allOptions}
+              onChange={handleFilterChange}
+              value={selectedOptions}
+              data-test-package-filter-select
+              id="packageFilterSelect"
+              ariaLabelledBy={labelId}
+              aria-label={<FormattedMessage id="ui-eholdings.label.packages" />}
+              tether={{
+                constraints: [
+                  {
+                    to: 'window',
+                    attachment: 'together',
+                  }
+                ]
+              }}
+            />
+          </Accordion>
+          <SearchFilters
+            searchType={searchTypes.PACKAGES}
+            activeFilters={searchFilters}
+            availableFilters={[
+              selectionStatusFilterConfig,
+            ]}
```

Comment:
Didn't see any reason why it should be written in separate strings
```suggestion
            availableFilters={[selectionStatusFilterConfig]}
```

Reply:
Fixed